.github/workflows/build-ci-buildbox-images.yaml

```diff
@@ -5,7 +5,9 @@ on:
     paths:
       - .github/workflows/build-ci-buildbox-images.yaml
       - build.assets/Dockerfile
+      - build.assets/Dockerfile-arm
```

Comment:
Drone builds its own buildbox images that are separate from these. I assume we want to be moving to buildbox images within GitHub Packages (as opposed to ECR), so we'll need this moving forward.

Also, for Linux `arm` and `arm64`, we build and use a completely separate set of Linux buildbox images in `teleport.e`. See `teleport.e/.github/workflows/build-linux-generic.yml`.

Reply:
In the repo we have

- _build.assets/Dockerfile_
- _build.assets/Dockerfile-arm_
- _build.assets/Dockerfile-centos7_
- _build.assets/Dockerfile-centos7-fips_
- **build.assets/Dockerfile-centos7-assets**
- **build.assets/Dockerfile-connect**
- **build.assets/Dockerfile-grpcbox**

with the **bolded** ones not being included in this file. I understand **Dockerfile-grpcbox** is only needed for local development, since we check in the generated grpc files, but why are we leaving out **Dockerfile-centos7-assets** and **Dockerfile-connect**?